src/components/ec-filter-popover/ec-filter-popover.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+  <div
+    class="ec-filter-popover"
+    data-test="ec-filter-popover"
+  >
+    <ec-popover
+      popover-class="ec-popover"
+      placement="bottom"
+      trigger="click"
+      level="modal"
+      @update:open="clickTrigger"
+    >
+      <div class="ec-filter-popover__filter-item">
+        <a class="ec-filter-popover__label">{{ label }}</a>
+        <!-- <ec-badge-number size="20" value="numberOfSelectedFilters" />
+        TODO Badge with https://fxsolutions.atlassian.net/browse/ONL-4909 -->
```

Comment:
For initial implementation, we can add 
```
<span v-if="numberOfSelectedFilters > 0">({{ numberOfSelectedFilters }})</span>
```

So we can test numberOfSelectedFilters prop in specs, in SB story

Reply:
I'v added the span with some styling - but we'd have to remove this styling once we work on ONL-4909. Is that ok?

Reply:
IMO that's fine, we'll pick it up from there